README.md

```diff
@@ -46,9 +48,11 @@ Component rendered on any tool part of topcoder (eg. platform-ui.topcoder.com/le
 ```
   tcUniNav('init', 'navigation-el', {...config, type: 'tool', toolName: 'Topcoder Academy'})
   tcUniNav('update', 'navigation-el', {user})
-  tcUniNav('update', 'navigation-el', {toolName: 'Work Manager})
+  tcUniNav('update', 'navigation-el', {toolName: 'Work Manager, tool})
 ```
 
+>**NOTE** In order to prevent flashing, the user info can only be set using the update method.
```

Comment:
@brooketopcoder this is not necessarily true. You can send the user at init if you have it. Eg. in wordpress, the templates are server rendered and there's no additional ajax to fetch user data (probably, depends on the implementation). We can send the user object when we initialize the uni nav.